web-app-router/pages/api/v1/verify/[app_id].ts

```diff
@@ -23,7 +23,7 @@ import {
 } from "@worldcoin/idkit-core";
 
 import * as yup from "yup";
-import { captureEvent } from "@/legacy/services/posthogClient";
+import { captureEvent } from "@/services/posthogClient";
```

Comment:
@andy-t-wang As I can see you've moved this helper to actual services folder, but still using only in legacy pages folder. If you need this helper in relevant code just duplicate it  to services and keep `@/legacy/services` for the `/pages`

Reply:
Ok duplicated it to the legacy folder